low-level/files/opt/seagate/sspl/setup/resource_map/lr2/resource_map.py

```diff
@@ -95,3 +79,24 @@ def set_health_data(health_data: dict, status, description=None,
             "recommendation": recommendation,
             "specifics": specifics
         })
+
```

Comment:
Why is manifest code in SSPL ? Major part of it should be in utils.

Reply:
We have added discovery API-related code in utils and the rest of the backend code we kept in SSPL for health as well as manifest. most of the code we have reused for health and manifest data from SSPL.

Reply:
Yes. @ujjwalpl  this is backend enabler serving manifest data, SSPL here.